homeassistant/components/squeezebox/strings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "Logitech Squeezebox",
+  "config": {
+    "step": {
+      "user": {
+	"title": "Configure Logitech Media Server",
+	"data": {
+	  "host": "Host (optional - leave blank to discover)"
+	}
+      },
+      "edit": {
+        "title": "Edit connection information",
+        "data": {
+          "host": "Host",
```

Comment:
Please reformat the json and use common keys here

`git grep 'key:common'`

Reply:
Got it. Added the common keys I could find.

Reply:
What is the proper way to update translations/en.json? I ran `python3 -m script.translations develop` but it did not update the common keys.

Reply:
I usually do it manually, but this might work:
`python -m script.translations develop --integration squeezebox`

Reply:
Thanks, I gave it a try but I guess the script doesn't handle these keys. I updated manually.